dbt/include/duckdb/macros/materializations/external.sql

```diff
@@ -62,10 +67,11 @@
   {{ adapter.rename_relation(intermediate_relation, target_relation) }}
 
   {{ run_hooks(post_hooks, inside_transaction=True) }}
-
+  -- What is should_revoke?
   {% set should_revoke = should_revoke(existing_relation, full_refresh_mode=True) %}
   {% do apply_grants(target_relation, grant_config, should_revoke=should_revoke) %}
 
+  --What does that do?
```

Comment:
This persists table/column comments if they are defined for the relation; DuckDB does not yet have this concept. See also: https://github.com/dbt-labs/dbt-adapters/blob/main/dbt/include/global_project/macros/adapters/persist_docs.sql

Reply:
aha, it pushes the information from the yaml files into the dwh and this is per adapter, i will also skip this one for the simplicity for now but will come back to that